decentraland/unity-explorer
# Feat: sdk7 video events

- add support for sdk video events
- removed duplication between Audio and Video streaming by merging it in one Media streaming component
- refactored Media Streaming code base to bi more align with ECS approach

# How to test
check that Video Streaming is not broken (see [this pr](https://github.com/decentraland/unity-explorer/pull/275) for test steps)
